fastlabel/__init__.py

```diff
@@ -3914,27 +3914,59 @@ def delete_dataset(self, dataset_id: str) -> None:
 
     # Dataset Object
 
-    def find_dataset_object(self, dataset_id: str, object_name: str) -> dict:
+    def find_dataset_object(
+        self,
+        dataset_id: str,
+        object_name: str,
+        version: str = None,
+        revision_id: str = None,
+    ) -> dict:
         """
         Find a dataset object.
+
+        dataset_id is dataset id (Required).
+        object_name is dataset object name (Required).
+        version is dataset version (Optional).
+        revision_id is dataset rebision (Optional).
+        Only use specify one of revision_id or version.
         """
-        endpoint = "datasets-v2/" + dataset_id  + "/objects/" + object_name
-        return self.api.get_request(endpoint)
+        if version and revision_id:
+            raise FastLabelInvalidException(
+                "only use specify one of revisionId or version.", 400)
+        endpoint = "datasets-v2/" + dataset_id + "/objects/" + object_name
+        params = {}
+        if revision_id:
+            params["revisionId"] = revision_id
+        elif version:
+            params["version"] = version
+        return self.api.get_request(endpoint, params=params)
 
     def get_dataset_objects(
-        self, dataset: str, version: str = None, tags: List[str] = []
+        self,
+        dataset: str,
+        version: str = None,
+        tags: List[str] = [],
```

Comment:
```suggestion
        tags: List[str] = None,
```
pythonにおいて、mutableなオブジェクトをデフォルト引数に与えると、副作用が発生する可能性があるので、Noneを指定して、関数内で tags = tags or [] という形で初期化する形で記述お願いします！


参考
https://qiita.com/7of9/items/5893e79819613d7f3c17

Reply:
対応済みです
(非常に勉強になります！！ありがとうございます。)